Makefile

```diff
@@ -12,6 +12,11 @@ all: $(TARGETS) ;
 
 $(TARGETS): cmake-build
 
+# Run CMake always with CMAKE_ARGS defined.
+ifdef CMAKE_ARGS
+.PHONY: $(BUILDDIR)/Makefile
+endif
+
```

Comment:
Stupid/inefficient with exported CMAKE_ARGS like on Travis.

But could be made to work only for `make … CMAKE_ARGS=…` (by looking at the origin of the var).

Reply:
Done.